### ISSUE TITLE
Add a light background behind "dark text" emojis

### ISSUE THREAD

**Issue by Artoria2e5** [tootsuite/mastodon#5083]
As noted in #2475, Mastodon's EmojiOne 2 seems to assume a light background, making a bunch of emojis unreadable. This not only applies to nonsense emoji representations for ©️ ®️ ™️, but also messes up, among others, 🔜. Since we now have full control of what emoji images we are using, it makes sense to go ahead and put a light background behind those troublesome images.

Glyphs to edit: 🎵🎶➕➖➗✖️💲💱™️©️®️〰️➰🔚🔙🔛🔝🔜✔️☑️⚫🔳▪️◾◼️⬛♠️♣️🏴🎱🎮🌑🖋️✒️💣📸📷. Inherently black things (geometric and playing cards stuff) can be made as bright as 👤. Clock symbols and silhouettes appear OK.

(For some reason I believe 👁️‍🗨️ is not working properly.)
See also EmojiTwo/emojitwo#110.

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [x] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases), 1.6.1.

**Comment by nightpool** [comment]
Does this happen with Twemoji, which we just switched to?

**Comment by Artoria2e5** [comment]
The arrows, arithmetic signs, and those "always black" things still appear problematic. Camera and bomb can use some outline.

**Comment by MightyPork** [comment]
A quick (though not extremely pretty) fix would be to use `filter: drop-shadow(0 0 1px white);`

®, © and ™ were excluded from emojification in the previous implementation, that should be added back imo. 

![screenshot_20170925_101412](https://user-images.githubusercontent.com/2041118/30798558-605c45ca-a1da-11e7-97e5-fd6346272921.png)

**Comment by Artoria2e5** [comment]
Yep, the fix to #2475 did remove emojificaton for those three nonsense characters.

**Comment by MightyPork** [comment]
it actually wasn't fixed until I made this PR in August: https://github.com/tootsuite/mastodon/pull/4472
Testing on glitch.social suggests the fix is still in place… the other symbols aren't so easy to work around tho :(

**Comment by wxcafe** [comment]
the `filter: drop-shadow(0 0 1px white)` is completely unreadable for me.

**Comment by renatolond** [comment]
I still see this issue. Following the suggestion on https://github.com/twitter/twemoji/issues/62 I've tried adding `filter: drop-shadow(1px 1px 0 #FFF) drop-shadow(-1px 1px 0 #FFF) drop-shadow(1px -1px 0 #FFF) drop-shadow(-1px -1px 0 #FFF);` and it looks quite good, but it breaks the margins
![image](https://user-images.githubusercontent.com/173791/35681547-02082248-075e-11e8-8fe5-5e5f16e72494.png)
![image](https://user-images.githubusercontent.com/173791/35681563-0e3943ee-075e-11e8-8dd2-c039dfbaf9fa.png)
